regolith/builders/recentcollabsbuilder.py

```diff
@@ -47,7 +47,16 @@ def construct_global_ctx(self):
     def latex(self):
         rc = self.rc
         since_date = dt.date.today() - relativedelta(months=48)
-        person = fuzzy_retrieval(all_docs_from_collection(rc.client, "people"), ['aka', 'name', '_id'], self.rc.people, case_sensitive = False)
+        if self.rc.people:
```

Comment:
in principle, people could be a list or a string, depending on if one more more than one person was given on the command line.  I think we want to just support one person for now, but I guess we should catch the case that multiple people are specified and raise an error with a slightly different error message?